### ISSUE TITLE
Bundle error with newest Kaminari version

### ISSUE THREAD

**Issue by sphinxc0re** [sferik/rails_admin#2823]
When bundling `rails_admin` with Kaminari (>= 1.0.1, ~> 1.0) I'm getting the following error:
```
Fetching gem metadata from https://rubygems.org/..........
Fetching version metadata from https://rubygems.org/..
Fetching dependency metadata from https://rubygems.org/.
Resolving dependencies...
Bundler could not find compatible versions for gem "kaminari":
  In Gemfile:
    kaminari (>= 1.0.1, ~> 1.0)

    rails_admin (~> 1.1.1) was resolved to 1.1.1, which depends on
      kaminari (~> 0.14)
```

**Comment by swistaczek** [comment]
Geting the same error.

**Comment by mshibuya** [comment]
Resolved by #2810.
